.github/workflows/smoke_test_assets/test.c

```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
```

Comment:
❓ Is the `smoke_tests` thing in any way docker specific? I don't necessarily think so, but if it is the name should reflect that.

Reply:
Well, it's being used only for testing the docker image soundness for now, but there's nothing that excludes any other usage.